rdmo/core/settings.py

```diff
@@ -133,8 +134,6 @@
 
 USE_I18N = True
 
-USE_L10N = True
```

Comment:
This settings will default to True in Django 4 and be removed in Django 5. So I decided to remove it already.